app/filters/api/authentication_filter.rb

```diff
@@ -0,0 +1,16 @@
+module API
+  class AuthenticationFilter
+    MAS_CMS_API_TOKEN = ENV.fetch('MAS_CMS_API_TOKEN')
```

Comment:
Good point. I would say the same as frontend behaviour. The same comment: https://github.com/moneyadviceservice/frontend/pull/1599#discussion_r88065107